jenkins/ocp-config/bc.yml

```diff
@@ -70,7 +70,10 @@ objects:
       type: Docker
     triggers: []
 parameters:
+# repository base url. In case bitbucket is used, needs to include /scm - so it's the path in front on opendevstack/..
```

Comment:
You can add them directly into the YML by using the `description` key.

Reply:
was not sure. we have both styles. but I would also prefer the description, unless we have multiple lines. So, I would propose: Include a (short) description and add further information as comments

Reply:
Sounds good.